src/Our.Umbraco.BulkUserAdmin/Web/UI/App_Plugins/BulkUserAdmin/js/bua.controllers.js

```diff
@@ -35,28 +39,28 @@
 	        });
 	    };
 
-	    $scope.isSelected = function(usr) {
+	    $scope.isSelected = function (usr) {
 	        return _.find($scope.selectedUsers, function (itm) {
 	            return itm.Id == usr.Id;
 	        }) !== undefined;
 	    };
 
 	    $scope.isSelectedAll = function () {
-	        return $scope.users !== undefined && _.every($scope.users.items, function(itm) {
-	            return !!_.find($scope.selectedUsers, function(itm2) {
+	        return $scope.users !== undefined && _.every($scope.users.items, function (itm) {
+	            return !!_.find($scope.selectedUsers, function (itm2) {
 	                return itm.Id == itm2.Id;
 	            });
 	        });
 	    };
 
 	    $scope.selectUser = function (usr) {
-            if (!$scope.isSelected(usr)) {
-                $scope.selectedUsers.push(usr);
-            } else {
-                $scope.selectedUsers = _.reject($scope.selectedUsers, function (itm) {
-                    return itm.Id == usr.Id;
-                });
-            }
+	        if (!$scope.isSelected(usr)) {
```

Comment:
Hmmm, good question. Like I say, no biggie, it's fixable.

Personally I tend to do a `CTRL+K,CTRL+D` in VS Studio to format the code, not sure if VSCode supports that? or maybe we could give something like [EditorConfig](http://editorconfig.org/) a try?

But I guess EditorConfig is something that we could apply on multiple projects for consistency.